benchmark.md

```diff
@@ -91,7 +95,6 @@ and also allow users to have access to pretrained agents.*
 |qrdqn|BeamRiderNoFrameskip-v4    |  17122.941| 10769.997|10M        |        596483|           17|
 |qrdqn|BreakoutNoFrameskip-v4     |    393.600|    79.828|10M        |        579711|           40|
 |qrdqn|CartPole-v1                |    500.000|     0.000|50k        |        150000|          300|
-|qrdqn|EnduroNoFrameskip-v4       |   3231.200|  1311.801|10M        |        585728|            5|
```

Comment:
it looks like I forgot to push the benchmark log of that one... (I will do that soon)
Please do the same for the robotics envs (files are in `logs/benchmark/`, you may need `git add -f` for that)